kubectl-fdb/cmd/cordon.go

```diff
@@ -67,6 +66,14 @@ func newCordonCmd(streams genericclioptions.IOStreams) *cobra.Command {
 			if err != nil {
 				return err
 			}
+			useCustomLabel, err := cmd.Flags().GetBool("use-custom-label")
+			if err != nil {
+				return err
+			}
+			customLabel, err := cmd.Flags().GetString("custom-label")
```

Comment:
```suggestion
			customLabel, err := cmd.Flags().GetString("cluster-label")
```

Why should we name it `custom-label` and not `cluster-label`?

Reply:
The idea is to allow other labels as well as per user requirement. 
Eg:
```
 kubectl fdb cordon -l "fdb-cluster-name some-other-label" node-1
 ```